packages/components/organisms/f-offers/src/components/Offers.vue

```diff
@@ -2,36 +2,30 @@
     <div
         :class="$style['c-offers']"
         data-test-id="offers">
-        <offers-search-box />
+        <no-results />
```

Comment:
If feels little confusing here as the file name suggests Offers, but those seem to have been replaced.

Reply:
No results is a component that is shown on the offers page (where the cards would appear) at all times. So I agree the `NoOffersFound` maybe a better name, thanks

Reply:
changed to your suggestion